src/basic/strv.c

```diff
@@ -458,6 +476,10 @@ int strv_split_and_extend_full(char ***t, const char *s, const char *separators,
         return (int) strv_length(*t);
 }
 
+int strv_split_and_extend(char ***t, const char *s, const char *separators, bool filter_duplicates) {
+        return strv_split_and_extend_full(t, s, separators, filter_duplicates, 0);
+}
```

Comment:
why can't this stay static inline? it doesn't require anything external?

Reply:
extract-word.h because the full definition of the enum is required